### PR TITLE
Mp 2459 axios client mocked mode

### PR DESCRIPTION
**Pull request by chopachop**
## What is the purpose of this pull request?
Implement Axios mocked environment for future implementation of the mocked requests, etc

## What has changed?
added required files for real and mocked Axios environment

## Notes for reviewer
I ran Prettier so the majority of changes are from it. Look at the files that were added in the PR to see what's new 

## Jira links
```jira_links
https://uniwise.atlassian.net/browse/MP-2459
```

### CODE REVIEW

File: dashboard/src/views/Projects/index.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  const [projectsList, setProjectsList] = useState<getProjectsResponse>(
+    mockedProjectsResponse,
+  );
+
+  useEffect(() => {
+    if (searchBar === "") {
+      return;
+    }
+    setProjectsList(projectsList);
+  }, [projectsList, searchBar]);
```
Comment (rimonnassory) [inline dashboard/src/views/Projects/index.tsx:22]: this could cause an infinite loop

**Review by rimonnassory — CHANGES_REQUESTED** [review]
I think overall it looks fine. 

I want you to look at the inline css and move it out to its own file.

File: dashboard/src/views/Projects/index.tsx (hunk 1 above)
Comment (chopachop) [reply dashboard/src/views/Projects/index.tsx:22]: I was concerned about that as well but getting a warning that I'm missing a dependency if I'm deleted either of them

**Comment by chopachop** [comment]
> I think overall it looks fine.
> 
> I want you to look at the inline css and move it out to its own file.

Due to custom way of styling in JoyUI that's used here I can't easily move it out in a separate file

File: dashboard/src/views/Projects/index.tsx (hunk 1 above)
Comment (rimonnassory) [reply dashboard/src/views/Projects/index.tsx:22]: i dotn think you should delete them, we just need to look at the prev and only update if there is a change. We can look at this tomorrow.

**Pull request merged**